app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,30 @@
+class SessionsController < ApplicationController
+  attr_accessor :user
+
+  before_action :load_user, only: :create
+
+  def new; end
+
+  def create
+    if @user&.authenticate params[:session][:password]
```

Comment:
```suggestion
    if @user.authenticate params[:session][:password]
```
ri dc nhé vì ở đây user chắc chắn khác nill rồi